java-extras/src/main/java/org/triplea/java/Sha512Hasher.java

```diff
@@ -0,0 +1,48 @@
+package org.triplea.java;
```

Comment:
I'd have gone with `org.triplea.crypto`, because this obviously is java code and I don't think it adds much value here.

Reply:
`org.triplea.hash` seems fine.
To my knowledge the `org.triplea.java` package only exist for this class, that's why I was suggesting to adjust the name to be more fitting for the classes - or rather class - inside

Reply:
Ahhh ok that makes sense then.
I must have missed this folder